### PR TITLE
Add delete method

### PR DESCRIPTION
**Pull request by robechon**
New API method to delete a server by id (with unit tests and examples)

### CODE REVIEW

**Comment by fwininger** [comment]
Can you rebase please ?

File: examples/delete_server_by_id.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+SERVER_ID = '' # For this example, you need to specify a server id in addition to your credentials
+
+result = CBWApi(API_URL, API_KEY, SECRET_KEY).delete_server_by_id(SERVER_ID)
+
+if result is True:
```
Comment (jinxka) [inline examples/delete_server_by_id.py]: I don't think it is useful to verify that True is True

File: cbw_api_toolbox/cbw_api.py [hunk 2]
```diff
@@ -86,3 +86,11 @@ def get_detailed_servers(self):
         for server in self.servers():
             result.append(self.server(server.server_id))
         return result
+
+    def delete_server_by_id(self, server_id):
+        """DELETE request to /api/v2/servers/SERVER_ID to delete a specific server by id"""
+        if server_id:
+            result = self._request("DELETE", [ROUTE_SERVERS, server_id])
+            if result and 'id' in result:
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_api.py]: Use the response code of the request.

File: cbw_api_toolbox/cbw_api.py [hunk 3]
```diff
@@ -86,3 +86,11 @@ def get_detailed_servers(self):
         for server in self.servers():
             result.append(self.server(server.server_id))
         return result
+
+    def delete_server_by_id(self, server_id):
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_api.py]: Rename to `delete_server`

File: examples/delete_server_by_id.py (hunk 1 above)
Comment (robechon) [reply examples/delete_server_by_id.py]: Done

File: cbw_api_toolbox/cbw_api.py (hunk 3 above)
Comment (robechon) [reply cbw_api_toolbox/cbw_api.py]: Done !

**Pull request closed without merging**